suse_migration_services/units/mount_system.py

```diff
@@ -109,7 +110,7 @@ def main():
     mount_system(root_path, fstab)
 
     initialize_logging()
-    initialize_debugging()
+    update_config_info()
```

Comment:
Can we name this

update_migration_config_file()

It would be more in line with the other config file related methods.